jest-common/src/test/java/io/searchbox/fields/FieldsStatsTest.java

```diff
@@ -16,25 +16,25 @@
 
     @Test
     public void testBasicUriGeneration() {
-        FieldStats fieldStats = new FieldStats.Builder(FIELDS).setIndex(INDEX).build();
```

Comment:
why we remove field stats tests?

Reply:
It looks like merge conflict from 6.3. I reverted FieldsStatsTest and added FieldCapabilitiesTest